src/styles/foundation/utilities.scss

```diff
@@ -1,62 +1,69 @@
-$default-browser-font-size: 16px;
-$base-font-size: 10px;
+$root-font-size: 16px;
 
 /// Returns the value in rem for a given pixel value.
 /// @param {Number} $value - The pixel value to be converted.
+/// @param {Number} $current-font-size - For converting ems, defines the pixel value of 1em.
 /// @return {Number} The converted value in rem.
 
-@function rem($value) {
+@function rem($value, $current-font-size: $root-font-size) {
   $unit: unit($value);
 
   @if $value == 0 {
     @return 0;
   } @else if $unit == 'rem' {
     @return $value;
   } @else if $unit == 'px' {
-    @return $value / $base-font-size * 1rem;
+    @return $value * (1rem / $root-font-size);
+    // e.g. 16px × 1rem/16px = 1rem
   } @else if $unit == 'em' {
-    @return $unit / 1em * 1rem;
```

Comment:
^ This looks like a bug. This code path (where `$value` is in `em` units) is an invalid Sass expression.

Reply:
(Probably intended to be `$value / 1em * 1rem`)

Reply:
`em` and `rem` are behaving so differently that I think we should `@error` when an `em` value is passed.

Reply:
What do you think of the updated implementation? It fixes this bug, I think

Reply:
The bug fix is valid!

I think we can safely stop allowing passing `em` values to the function:

1. This use-case didn't work before, therefore it's not a working part of the API
1. If people were using `rem(<number>em)`, they'd have raised an issue because it throws `Undefined operation: ""em"/1em times 1rem". on line x at column 13.`